hack/prom-metric-linter/metric_name_linter.sh

```diff
@@ -20,7 +20,7 @@
 
 set -e
 
-linter_image_tag="v0.0.1"
+linter_image_tag="v0.0.6"
```

Comment:
Out of curiosity, why you need to bump it? Is it related to https://github.com/kubevirt/monitoring/pull/244 ?

Reply:
Yes, exactly.